server/meshmodel/nginx-ingress/relationships/hierarchical_inventory.json

```diff
@@ -0,0 +1,79 @@
+{
+  "schemaVersion": "core.meshery.io/v1alpha2",
+  "version": "*",
+  "kind": "Hierarchical",
+  "metadata": {
+    "description": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. E.g Secret->Pod, ConfigMaps->Deployment, etc..."
+  },
+  "model": {
+    "schemaVersion": "core.meshery.io/v1beta2",
```

Comment:
I suspect there's some confusion here. While we need to upgrade to v1beta1 of the Relationships schema, you'll want to use v1alpha2 for the time being.